kong/core/router.lua

```diff
@@ -433,30 +433,38 @@ function _M.new(apis)
     categorize_api_t(api_t, categories)
   end
 
+  local compare = function(a,b, category_bit)
```

Comment:
style: missing a space after the first argument `a,`. We also favor declaration of local function with the following form:
```
local function compare(a, b, category_bit)

end
```